Frends.Community.Email/EmailTask.cs

```diff
@@ -156,7 +156,7 @@ public static async Task<Output> SendEmailToExchangeServer([PropertyTab] Exchang
             var graph = new GraphServiceClient(credentials);
 
             var encoding = GetEncoding(input.MessageEncoding);
-            var bytes = Encoding.Default.GetBytes(input.Message);
+            var bytes = encoding.GetBytes(input.Message);
```

Comment:
So, I am unsure what this bug is about exactly and what the root cause is, but I wonder if this is a good fix?.. I mean you are just converting a string to byte array and back (using the same encoding). Is this doing anything? If this is a correct fix also - should you just remove the whole byte conversion altogether?..

Reply:
Well, this is the only way I know how to switch the encoding. This way you basically force the encoding to some text. You can see that the encoding changes from unit tests https://github.com/CommunityHiQ/Frends.Community.Email/blob/ISSUE-31/Frends.Community.Email.Tests/SendExchangeEmailTests.cs#L152 and https://github.com/CommunityHiQ/Frends.Community.Email/blob/ISSUE-31/Frends.Community.Email.Tests/SendExchangeEmailTests.cs#L174.